stacks-common/src/types/mod.rs

```diff
@@ -431,6 +431,12 @@ impl SIP031EmissionInterval {
 }
 
 impl StacksEpochId {
+    #[cfg(any(test, feature = "testing"))]
+    pub const fn latest() -> StacksEpochId {
+        StacksEpochId::Epoch33
+    }
+
+    #[cfg(not(any(test, feature = "testing")))]
```

Comment:
This seems like a footgun IMO. What about using `TestFlag` to overwrite this just for the integration tests that need it?

Reply:
I'm also curious about this.  Why do we ever want `StacksEpochId::latest()` to be anything other than the actual latest epoch?  Like, how hard would it be to add `StacksEpochId::next()` to return the under-development epoch version, for the code paths that need it?